docs/docs/modules/indexes/document_loaders/examples/web_loaders/notiondb.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Can you change this to an `.mdx` file and move the example code to the `examples/` directory? 

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md